tests/type-check/main.arr

```diff
@@ -58,38 +66,38 @@ end
 
 check "These should all be good programs":
   base = "./tests/type-check/good/"
-  good-progs = FL.list-files(base)
+  # good-progs = FL.list-files(base)
+  good-progs = [list: "_plus.arr"]
   for each(prog from good-progs):
     when is-arr-file(prog) block:
-      filename = base + prog
-      result = compile-str(filename)
+      result = run-typed-file(base, prog)
```

Comment:
You're not using `compile-file` above now, you're using `run-typed-file` which seems to be unrelated...

Reply:
The issue is that `compile-file` doesn't actually run the compiled code. `run-typed-file` is my attempt to actually run the good type checker tests.